core/scripts/sql/texera_ddl.sql

```diff
@@ -19,16 +19,20 @@ SET GLOBAL time_zone = '+00:00'; # this line is mandatory
 
 CREATE TABLE IF NOT EXISTS user
 (
-    `name`      VARCHAR(32)                 NOT NULL,
-    `uid`       INT UNSIGNED AUTO_INCREMENT NOT NULL,
-    `password`  VARCHAR(256),
-    `google_id` VARCHAR(256) UNIQUE,
-    PRIMARY KEY (`uid`),
+    `email` varchar(256) NOT NULL,
+	`uid` int unsigned NOT NULL AUTO_INCREMENT,
+    `name` varchar(256),
+	`password` varchar(256) DEFAULT NULL,
+	`google_id` varchar(256) DEFAULT NULL,
+	`permission` int NOT NULL DEFAULT '0',
```

Comment:
Also please format this script properly. It seems those fields are not at the same indentation. This may create unnecessary version control diffs in future PRs.

Reply:
I will exclude permission from this PR.

Reply:
Thanks. Please modify PR title and PR description accordingly. Maybe `Introduce email field in user table`